packages/github/lib/clients.js

```diff
@@ -1,15 +1,33 @@
 const { createApolloFetch } = require('apollo-fetch')
+const Bottleneck = require('bottleneck')
+const debug = require('debug')('github:clients')
+
 const GitHubApi = require('@octokit/rest')
+
 const appAuth = require('./appAuth')
 
 const {
+  GITHUB_GRAPHQL_RATELIMIT,
   GITHUB_LOG_RATELIMIT
 } = process.env
+
 const DEV = process.env.NODE_ENV && process.env.NODE_ENV !== 'production'
 
 let installationToken
 let nextRateLimitCheck
 
+const limiter = new Bottleneck({
+  maxConcurrent: 1,
+  minTime: 200
```

Comment:
```suggestion
  minTime: 100
```
Isn't 200 a little bit too slow?

Reply:
Done so. Several test ran flawlessly.